### PR TITLE
Replace KResponseWindow with useKResponseWindow in Device plugin

### LINKED ISSUE #11322: Replace KResponsiveWindow mixin by useKResponsiveWindow composable - Device plugin
This is one of the issues whose common goal is to gradually replace `KResponsiveWindow` mixin by `useKResponsiveWindow` composable.

First, see https://github.com/learningequality/kolibri/issues/11321 for detailed instructions and acceptance criteria common to all issues.

Then, replace `KResponsiveWindow` mixin by `useKResponsiveWindow` composable in all .vue files of **the Device plugin (/kolibri/plugins/device)**

### PR DESCRIPTION
**Pull request by thesujai**
## Summary

I searched for all .vue in `kolibri/plugins/device`, and i started to replace `KResponsiveWindow` with `useKResponsiveWindow`

**For testing:**

1. First Identified where ResponsiveWindow elements are are used
2. Then play with CSS class of components that will be used when `windowBreakpoint` is reached.
3. Then verify that the CSS tweaks is consistent with both `KResponsiveWindow` and `userKResponsiveWindow`
4. Roll back the CSS tweaks made while testing

**Page Affected**

- en/device/#/content
- en/device/#/content/manage_tasks
- en/device/#/content/manage_channel
- en/device/#/content/channels
- en/device/#/facilities
- en/device/#/info

…

## References

Fixes #11322

## Reviewer guidance

Would it be possible to add a `hacktoberfest-accepted` label in this PR?
Here is the link: https://hacktoberfest.com/participation/#maintainers
Its fine if for some reasons you aren't able to do so:)
…

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by thesujai** [comment]
@MisRob please have a look

File: kolibri/plugins/device/assets/src/views/ManageTasksPage/TaskPanel.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
 
-  <div class="task-panel" :class="{ 'task-panel-sm': windowIsSmall }">
+  <div class="task-panel" :class="{ 'task-panel-sm': windowWidth<600 }">
```
Comment (thesujai) [inline kolibri/plugins/device/assets/src/views/ManageTasksPage/TaskPanel.vue]: `windowIsSmall` is only working with `KResponsiveWindow`, not with `userKResponsiveWIndow`.
But this issue was with only few pages

Comment (MisRob) [reply kolibri/plugins/device/assets/src/views/ManageTasksPage/TaskPanel.vue]: I can see that at some places you import `windowIsSmall` in this manner:

```js
const windowIsSmall = useKResponsiveWindow();
```

It is supposed to be 

```js
const { windowIsSmall } = useKResponsiveWindow();
```

See "Usage -> As composable" section here https://develop--kolibri-design-system.netlify.app/kresponsivewindow/#usage

Can you try out if `windowIsSmall`} works as expected after correcting this?

Comment (thesujai) [reply kolibri/plugins/device/assets/src/views/ManageTasksPage/TaskPanel.vue]: Yes, it does work. Thank You

**Comment by MisRob** [comment]
@thesujai Thanks! I will review code in more detail later on, but briefly skimmed through it and looks great.

Can you recall which pages are affected? For our QA team, it'd be helpful to have a list of pages from the user-point of view, for example their URLs

**Comment by MisRob** [comment]
^ This would also helps us in your other PRs

**Comment by MisRob** [comment]
Here's a nice example https://github.com/learningequality/kolibri/pull/11346#issuecomment-1746595012

**Comment by thesujai** [comment]
Yes sure

**Comment by thesujai** [comment]
@MisRob For few changes which I did, I was not able to visually find the page in which it is reflected due to my less familiarity with the codeabse. Like file `device/assets/src/views/ManageContentPage/ChannelPanel/ChannelDetails.vue` and `device/assets/src/views/SelectContentPage/index.vue`.

**Comment by MisRob** [comment]
Thank you @thesujai, it's okay especially when you mentioned the places that you couldn't locate. I will try it out.

**Comment by MisRob** [comment]
@thesujai Could you please rebase on top of the latest develop? That should hopefully resolve failing test check.

**Comment by thesujai** [comment]
Hey @MisRob, can say a suitable method for rebasing it, last time i tried in another pr and it got messed up

**Comment by MisRob** [comment]
@thesujai Here are the guidelines https://github.com/learningequality/kolibri/issues/11361.

However, let's wait, we figured out meanwhile that there's more things that need to be fixed so rebase wouldn't help at this point. I will review and eventually rebase this later on.

**Comment by thesujai** [comment]
Okay @MisRob

**Review by MisRob — APPROVED** [review]
Great work @thesujai.

Code is fine, I tested all the places and haven't observed any regressions or console errors.

**Comment by MisRob** [comment]
Let's hold merging though for some time. As soon as tests are fixed on `develop`, I will rebase this PR and merge it.

**Pull request merged**